src/gui/modals/EffectSelectDialog.cpp

```diff
@@ -126,7 +127,12 @@ EffectSelectDialog::EffectSelectDialog(QWidget* parent) :
 
 	m_filterEdit = new QLineEdit(this);
 	connect(m_filterEdit, &QLineEdit::textChanged, this, [this](const QString &text) {
+// TODO: Cleanup when we don't support Qt5 anymore
+#if (QT_VERSION >= QT_VERSION_CHECK(5,12,00))
```

Comment:
```suggestion
#if (QT_VERSION >= QT_VERSION_CHECK(5,12,0))
```
`00` is actually an octal integer literal (yay C++). It wouldn't misbehave in this case, but it's still better to avoid